apiserver/facades/client/bundle/bundle.go

```diff
@@ -335,25 +337,30 @@ func bundleOutputFromBundleData(bd *charm.BundleData) *bundleOutput {
 	}
 }
 
-func (b *BundleAPI) fillBundleData(model description.Model, includeCharmDefaults bool, backend Backend) (*charm.BundleData, error) {
+func (b *BundleAPI) fillBundleData(model description.Model, includeCharmDefaults, includeSeries bool, backend Backend) (*charm.BundleData, error) {
 	cfg, err := config.New(config.NoDefaults, model.Config())
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
-	var base corebase.Base
+	var defaultBase corebase.Base
 	value, ok := cfg.DefaultBase()
 	if ok {
 		var err error
-		base, err = corebase.ParseBaseFromString(value)
+		defaultBase, err = corebase.ParseBaseFromString(value)
 		if err != nil {
 			return nil, err
 		}
 	} else {
-		base = version.DefaultSupportedLTSBase()
+		defaultBase = version.DefaultSupportedLTSBase()
 	}
-	defaultSeries, err := corebase.GetSeriesFromBase(base)
-	if err != nil {
-		return nil, err
+
+	var defaultSeries string
+	if includeSeries {
+		var err error
+		defaultSeries, err = corebase.GetSeriesFromBase(defaultBase)
+		if err != nil {
+			return nil, err
+		}
```

Comment:
IMO, cleaner to put this logic inside the if statement where the value of `defaultSeries` is actually needed (i.e. replacing line 374)

Reply:
Ah yes, since it's only used once in this function makes sense

Reply:
Yes fair. Since `defaultSeries` is only used once in this function